src/styles/cloudColors.ts

```diff
@@ -40,8 +40,7 @@ export const saturatedTeal = '#36c7c4';
 export const saturatedBlue = '#378fee';
 export const saturatedLavender = '#625fe3';
 
-// export the variables for javascript access
-export default {
+const CLOUD_COLORS = {
```

Comment:
can you rename this constant to `LAKEFRONT_COLORS` instead?

Reply:
@amirkibbar - Done. I went ahead and renamed the file across the library as well.